pode.build.ps1

```diff
@@ -422,15 +422,22 @@ Task DocsBuild DocsDeps, DocsHelpBuild, {
 
 Task Clean  {
     $path = './deliverable'
-    if (Test-Path $path) {
+    if (Test-Path -Path $path -PathType Container) {
         Remove-Item -Path $path -Recurse -Force | Out-Null
     }
 
     $path = './pkg'
 
-    if (  (Test-Path $path)) {
+    if ((Test-Path -Path $path -PathType Container )) {
         Remove-Item -Path $path -Recurse -Force | Out-Null
     }
+
+    if ((Test-Path -Path .\packers\choco\tools\ChocolateyInstall.ps1 -PathType Leaf )) {
+        Remove-Item -Path .\packers\choco\tools\ChocolateyInstall.ps1
+    }
+    if ((Test-Path -Path .\packers\choco\pode.nuspec -PathType Leaf )) {
+        Remove-Item -Path .\packers\choco\pode.nuspec
+    }
```

Comment:
Do not delete these files, they're important.

Reply:
those files are created using the `/pode_template.nuspec` file.
previously the build was changing the version on `pode_template.nuspec` directly compromising any futher build with a different version

Line 135 -   (Get-Content ./packers/choco/pode_template.nuspec) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./packers/choco/pode.nuspec

Reply:
Apologies, I didn't see the `_template` files. Marking as resolved.